script/Arcueid/Day 8/08_00_ARC08_1_1.txt

```diff
@@ -38,13 +38,13 @@ If anything, it seems Akiha's gaze got even sharper.
 [sha:a7f6007faac8f2a39a28d654cfa224196fce5458]{
 -- Page 36, Offset 19766.
 -- ……辛いな。
-...What a bind.
+...This is bad. //Changed it but not sure if my reasoning is factual. Usually when I see a saying with "bind" it's usually a situation where the person is presented with two options/factors and either way they're fucked hence them being in a bind. Not sure on it though.
```

Comment:
I interpret bind as an unpleasant situation that's difficult to extricate yourself from, though choice is definitely another use of it. 

That said, now that I look at this I'm not sure if tsurai here is actually supposed to be referring more directly to Akiha's harshness? In a "Damn, she's tough..." sorta way